### PR TITLE
fix: Improving error messages

### PR DESCRIPTION
**Pull request by aliehsaeedii**
Solves issue #8952 
### Description 
Creating a source on an already existing Kafka topic with a defined schema requires adding VALUE_SCHEMA_ID in the WITH clause. Otherwise INSERT INTO fails with an ambiguous error message as explained [here](https://github.com/confluentinc/ksql/issues/8952). This fix adds a hint to the error message for an easier debugging.

### Testing done 
Manual Testing
QTT is not able to cover this case.

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: ksqldb-parser/src/main/java/io/confluent/ksql/parser/SyntaxErrorValidator.java [hunk 1]
```diff
@@ -143,7 +143,7 @@ public void syntaxError(
         final String tokenName = ((Token) offendingSymbol).getText();
         sb.append(
             String.format("\"%s\" is a reserved keyword and it can't be used as an identifier."
-                + " You can use it as an identifier by escaping it as '%s' ",
+                    + " You can use it as an identifier by escaping it as '%s' ",
```
Comment (jnh5y) [inline ksqldb-parser/src/main/java/io/confluent/ksql/parser/SyntaxErrorValidator.java:146]: Are the only changes in this file whitespace?

File: ksqldb-parser/src/test/java/io/confluent/ksql/parser/KsqlParserTest.java [hunk 2]
```diff
@@ -484,8 +484,8 @@ public void shouldAllowEscapedTerminateQuery() {
   public void testSelectAllJoin() {
     // When:
     final Query query = KsqlParserTestUtil.<Query>buildSingleAst(
-        "SELECT * FROM test1 t1 LEFT JOIN test2 t2 ON t1.col1 = t2.col1 WHERE t2.col2 = 'test';",
-        metaStore)
+            "SELECT * FROM test1 t1 LEFT JOIN test2 t2 ON t1.col1 = t2.col1 WHERE t2.col2 = 'test';",
```
Comment (jnh5y) [inline ksqldb-parser/src/test/java/io/confluent/ksql/parser/KsqlParserTest.java:487]: Are the only changes in this file whitespace?

**Comment by jnh5y** [comment]
Hmm... the `elements.json` QTT seems to get output from `KsqlConnectSerializer.java`.

Did you try updating that particular QTT?

**Comment by aliehsaeedii** [comment]
> Hmm... the `elements.json` QTT seems to get output from `KsqlConnectSerializer.java`.
> 
> Did you try updating that particular QTT?

I just tried it and unfortunately it could not catch the exception as well :(

**Review by jnh5y — APPROVED** [review]
@aliehsaeedii Looks good.  I was able to add a QTT.  

Does that look reasonable?

**Review by jaceklaskowski — APPROVED** [review]
LGTM

**Pull request merged**